### ISSUE TITLE
ipa_replication_check - metrics disappeared after the version '0.0.10'

### ISSUE THREAD

**Issue by maxozerov** [camptocamp/ipahealthcheck_exporter#6]
**ipahealthcheck_exporter-0.0.12.linux-amd64**

```# HELP ipa_dogtag_connectivity_check Check to verify dogtag basic connectivity. (1: success, 0: error)
# TYPE ipa_dogtag_connectivity_check gauge
ipa_dogtag_connectivity_check{ipahealthcheck="DogtagCertsConnectivityCheck"} 1
# HELP ipa_service_state State of the services monitored by IPA healthcheck (1: running, 0: not running)
# TYPE ipa_service_state gauge
ipa_service_state{service="certmonger"} 1
ipa_service_state{service="dirsrv"} 1
ipa_service_state{service="gssproxy"} 1
ipa_service_state{service="httpd"} 1
ipa_service_state{service="ipa_custodia"} 1
ipa_service_state{service="ipa_dnskeysyncd"} 1
ipa_service_state{service="ipa_otpd"} 1
ipa_service_state{service="kadmin"} 1
ipa_service_state{service="krb5kdc"} 1
ipa_service_state{service="named"} 1
ipa_service_state{service="pki_tomcatd"} 1
ipa_service_state{service="sssd"} 1
```
---
**ipahealthcheck_exporter-0.0.9.linux-amd64** & **ipahealthcheck_exporter-0.0.10.linux-amd64**

```# HELP ipa_dogtag_connectivity_check Check to verify dogtag basic connectivity. (1: success, 0: error)
# TYPE ipa_dogtag_connectivity_check gauge
ipa_dogtag_connectivity_check{ipahealthcheck="DogtagCertsConnectivityCheck"} 1
# HELP ipa_replication_check Replication checks (1: success, 0: error)
# TYPE ipa_replication_check gauge
ipa_replication_check{ipahealthcheck="ReplicationChangelogCheck"} 1
ipa_replication_check{ipahealthcheck="ReplicationCheck"} 1
# HELP ipa_service_state State of the services monitored by IPA healthcheck (1: running, 0: not running)
# TYPE ipa_service_state gauge
ipa_service_state{service="certmonger"} 1
ipa_service_state{service="dirsrv"} 1
ipa_service_state{service="gssproxy"} 1
ipa_service_state{service="httpd"} 1
ipa_service_state{service="ipa_custodia"} 1
ipa_service_state{service="ipa_dnskeysyncd"} 1
ipa_service_state{service="ipa_otpd"} 1
ipa_service_state{service="kadmin"} 1
ipa_service_state{service="krb5kdc"} 1
ipa_service_state{service="named"} 1
ipa_service_state{service="pki_tomcatd"} 1
ipa_service_state{service="sssd"} 1
```

**Comment by maxozerov** [comment]
this was probably done to improve time-performance (i.e. reading from the 'healthcheck.log')

**Comment by taniahagan** [comment]
https://github.com/camptocamp/ipahealthcheck_exporter/pull/8 brings back the replication check

**Comment by lconsuegra** [comment]
Fixed in [0.0.13](https://github.com/camptocamp/ipahealthcheck_exporter/releases/tag/v0.0.13)
